internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
+{
+<#
+.SYNOPSIS
+Get a connection object from a given computer
+
+.DESCRIPTION
+Retrieves the IPAddress, ComputerName from one computer.
+The object can be used to take action against its name or IPAddress.
+
+First ICMP is used to test the connection, and get the connected IPAddress.
+
+If your local Powershell version is not higher than 2, WMI is tried to get the computername.

```

Comment:
If your local Powershell version is not higher than 2, 

The module is forbidden to run on versions lower than 3, as specified in the psd1. But please keep as many failbacks as possible.